.circleci/continue_config.yml

```diff
@@ -74,9 +74,25 @@ jobs:
       - store_artifacts:
           path: << parameters.project >>/build/reports/tests
 
+  noop:
+    docker:
+      - image: cimg/base:current
+    resource_class: small
+    steps:
+      - run: echo "noop"
+
 workflows:
   version: 2
-  build-test-and-deploy-model:
+  no-changes:
+    when:
+      not:
+        or:
+          - << pipeline.parameters.build-model >>
+          - << pipeline.parameters.build-indexer >>
+          - << pipeline.parameters.build-search >>
+    jobs:
```

Comment:
can't have an empty workflow - see https://github.com/CircleCI-Public/circleci-cli/issues/577.  Have to therefore add a no-op  job